### PR TITLE
[Fix-4090][dao] Add null judgment

### PR DESCRIPTION
**Pull request by liliang1991**
## What is the purpose of the pull request

FIX DependNode NPE 

## Verify this pull request

This pull request is already covered by existing tests, such as *(please describe tests)*.

### CODE REVIEW

**Comment by lenboo** [comment]
@liliang1991 
welcome your first commit. 
1. please change the title and content to english.
2. create an issue about the problem and contact this with (#issueNumber)
3. pr title should be [FIX-issueNumber][Module] xxxxx
4. you can find some other document about pr here : https://dolphinscheduler.apache.org/zh-cn/docs/development/pull-request.html

**Comment by liliang1991** [comment]
Add null judgment

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java [hunk 1]
```diff
@@ -292,7 +295,6 @@ public static boolean allDependsForbiddenOrEnd(TaskNode taskNode,
      * this function parse the condition node to find the right branch.
      * also check all the depends nodes forbidden or complete
      *
-     * @param preNodeName
```
Comment (zhuangchong) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java:295]: Why remove argument comments from a method？

**Comment by CalvinKirs** [comment]
Hi, as @zhuangchong   said, you can use plug-ins to automatically format the code, instead of directly deleting the code or comments that affect the format. 

you can find some other document about pr here : https://dolphinscheduler.apache.org/zh-cn/docs/development/pull-request.html

**Comment by liliang1991** [comment]
I use the official website plug-in to format the code


 原始邮件 
发件人: zhuangchong<notifications@github.com>
收件人: apache/incubator-dolphinscheduler<incubator-dolphinscheduler@noreply.github.com>
抄送: liliang1991<liliang_68@126.com>; Mention<mention@noreply.github.com>
发送时间: 2020年11月21日(周六) 19:13
主题: Re: [apache/incubator-dolphinscheduler] [Fix-4090][dao] Add nulljudgment (#4088)


@zhuangchong commented on this pull request.
In dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java:
> @@ -292,7 +295,6 @@ public static boolean allDependsForbiddenOrEnd(TaskNode taskNode, * this function parse the condition node to find the right branch. * also check all the depends nodes forbidden or complete * - * @param preNodeName
Why remove argument comments from a method？

**Review by lenboo — APPROVED** [review]
+1

**Review by CalvinKirs — APPROVED** [review]
+1

**Comment by CalvinKirs** [comment]
considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**